src/pkgcore/ebuild/repository.py

```diff
@@ -670,6 +673,27 @@ def deprecated(self):
             pkg_deprecated.update(repo.config.pkg_deprecated)
         return packages.OrRestriction(*pkg_deprecated)
 
+    @klass.jit_attr
+    def stabilization_groups(self):
+        """Return a mapping of stabilization groups to packages."""
+        stabilization_groups = {}
+        base_dir = pjoin(self.location, "metadata", "stabilization-groups")
+        for dirname, _dirs, files in os.walk(base_dir):
+            dirbase = dirname.removeprefix(base_dir)
+            for file in files:
```

Comment:
I'd rather you require a specific suffix. Otherwise we're going to end up with the mess of reading backup files accidentally.

Reply:
We don't have any suffix defined for those, but I do plan to verify this in pkgcheck with error level (so it breaks the QA bot). I prefer those names be as simple as possible.

Reply:
Just make them `*.group`, that should be simple and clean, and it will avoid issues in the future. We've made a lot of recursive directories in Portage with no suffixes, and you know how complex handling these has become.